app/views/katello/api/v2/systems/product_content.json.rabl

```diff
@@ -0,0 +1,17 @@
+object false
+
+extends "katello/api/v2/common/metadata"
+
+child @collection[:results] => :results do
+  child :product => :product do
+    attributes :id, :name
+  end
+
+  child :content => :content do
+    attributes :id, :name, :label
+  end
+
+  node :override do |pc|
+    pc.content_override(@system)
+  end
+end
```

Comment:
Do we need two separate templates? Looks like the code is the same as `content_override.json.rabl`.